src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -115,11 +115,52 @@ static View()
             RegisterAccessibilityDelegate();
         }
 
+        /// <summary>
+        /// Accessibility mode for controlling View's Accessible implementation.
+        /// It is only relevant when deriving custom controls from View directly,
+        /// as classes derived from CustomView (or any of its subclasses) get the
+        /// Custom mode by default.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public enum ViewAccessibilityMode
+        {
+            /// <summary>
+            /// Default accessibility implementation. Overriding View.Accessibility...()
+            /// virtual methods will have no effect.
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            Default,
+            /// <summary>
+            /// Custom accessibility implementation. Overriding View.Accessibility...()
+            /// will be necessary to provide accessibility support for the View.
+            /// </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
+            Custom,
+        }
+
+        private static IntPtr NewWithAccessibilityMode(ViewAccessibilityMode accessibilityMode)
+        {
+            switch (accessibilityMode)
+            {
+            default:
+            case ViewAccessibilityMode.Default:
+                return Interop.View.New();
+
+            case ViewAccessibilityMode.Custom:
+                return Interop.View.NewCustom();
```

Comment:
```suggestion
            case ViewAccessibilityMode.Custom:
            {
                return Interop.View.NewCustom();
            }
            case ViewAccessibilityMode.Default:
            default:
            {
                return Interop.View.New();
            }
```

Wrap by brace
+
Places 'default' as end of cases (It is my personal taste)

Reply:
Reordering done & braces added. Also indentation added so that the braces are all at different level.s